### DISCUSSION TITLE
"How to set 'num_workers' during training to speed up CPU data loading?"

### DISCUSSION THREAD

**Discussion by KevinWuang** [MIC-DKFZ/nnUNet#1783] [Q&A] [answered]
"My dataset is quite large, and the training speed is slow. How can I speed up the training by adjusting 'num_workers'? Are there any other methods? Can you provide more details?"

**Comment by ancestor-mithril** [comment, marked as answer]
Use the environment variables `nnUNet_n_proc_DA` (default: 8) to change the number of workers. 
It is not recommended to increase it if you have less than 64GB RAM. 
If you have less, the training might be faster if you decrease the number of workers.

**Reply by KevinWuang** [reply]
Thanks
